test/evergreen.yml

```diff
@@ -2136,6 +2136,40 @@ tasks:
           working_dir: "wiredtiger"
           command: bash test/evergreen/compatibility_test_for_wiredtiger_releases.sh
 
+  - name: format-stress-sanitizer-test-ppc
+    # Set 2.5 hours timeout (60 * 60 * 2.5)
+    exec_timeout_secs: 9000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          # CC is set to the system default "clang" binary here as a workaround. Change it back to mongodbtoolchain "clang" binary after BUILD-10248 fix.
+          configure_env_vars: CCAS=/opt/mongodbtoolchain/v3/bin/gcc CC=/usr/bin/clang CXX=/opt/mongodbtoolchain/v3/bin/clang++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-ggdb -fPIC -fsanitize=address -fno-omit-frame-pointer -I/opt/mongodbtoolchain/v3/lib/gcc/ppc64le-mongodb-linux/8.2.0/include"
```

Comment:
Not sure if CCAS setting makes any difference. Suggest to use the same `configure_env_vars` setting as function `compile wiredtiger address sanitizer`.

Reply:
The compilation fails if the CCAS setting is removed. 
I have retained CCAS setting and other options are set similar to  `compile wiredtiger address sanitizer`.